src/components/mapPageComponents/ducks/types.ts

```diff
@@ -46,18 +46,45 @@ interface NeighborhoodFeaturePropertiesResponse {
   lng: number;
 }
 
+// ---------------------------------Sites----------------------------------------
+
+export interface SiteGeoData {
+  type: string;
+  name: string;
+  features: SiteFeatureResponse[];
+}
+
+interface SiteFeatureResponse {
+  type: string;
+  properties: SiteFeaturePropertiesResponse;
+  geometry: MapGeometry;
+}
+
+interface SiteFeaturePropertiesResponse {
+  id: number;
+  treePresent: boolean;
+  diameter: number;
+  species: string;
+  updatedAt: string;
+  updatedBy: string;
+  address: string;
+  lat: number;
+  lng: number;
+}
+
 // ---------------------------------Shared Types----------------------------------------
 
 interface MapGeometry {
   type: string;
-  coordinates: Coordinate[][][];
+  coordinates: Coordinate[][][] | Coordinate;
```

Comment:
⛏️ This data type seems a little strange? Do you think you might want some tsdoc to explain why it can be a  List of List of List of Coordinate or just a coordinate? It seems a bit strange

Reply:
This is because GeoJSON has different [coordinates types](https://tools.ietf.org/html/rfc7946#section-3.1.1) depending on the geometry type, added a reference comment.